website/content/en/docs/reference/settings.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Settings"
+linkTitle: "Settings"
+weight: 5
+description: >
+  Configure Karpenter
+---
+
+Karpenter surfaces environment variables and CLI parameters to allow you to configure certain global settings on the controllers. These settings are described below.
+
+| Environment Variable | CLI Flag | Description |
```

Comment:
@matthchr  here is those docs you were asking about for the options of our controller